packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -302,8 +318,31 @@ export const createDefaultCallingHandlers = memoizeOne(
       await call?.removeParticipant(fromFlatCommunicationIdentifier(userId));
     };
 
+    /* @conditional-compile-remove(PSTN-calls) */
+    const onAddParticipant = async (
+      participant:
+        | CommunicationUserIdentifier
+        | PhoneNumberIdentifier
+        | UnknownIdentifier
+        | MicrosoftTeamsUserIdentifier,
+      options?: AddPhoneNumberOptions
+    ): Promise<void> => {
+      const newACSorTeamsUser = participant as CommunicationUserIdentifier | MicrosoftTeamsUserIdentifier;
+      const newPSTNUser = participant as PhoneNumberIdentifier;
```

Comment:
Wouldn't `if (newPSTNUser)` always be true?

Reply:
if the casting to that type fails it will be undefined since the properties don't align for those types, so when passing in a participant only one of the casts will work.

Reply:
The casting is compile time only and won't fail during run time.